_scicomputing/store_scratch.md

```diff
@@ -1,216 +1,29 @@
 ---
-title: Data Storage in Temporary Storage (Scratch)
-primary_reviewers: vortexing
+title: Scratch File Service
+primary_reviewers: scicomp
 ---
 
 
-`Scratch` storage serves as a temporary location for large data sets that ideally reside in an archive space like [AWS S3](/scicomputing/store_objectstore/) storage, to be transferred to when compute processes are applied to them.  Data in `Scratch` are typically then deleted automatically after certain timeframes when they are no longer needed. Intermediate data that is generated can be saved in `Scratch` as well, and then the final data resulting from the compute process can be written to [`Fast`](/scicomputing/store_posix/) storage for the researcher.  This allows large data to be archived in [AWS S3](/scicomputing/store_objectstore/) storage, accessed by HPC when it is temporarily housed in `Scratch` and only the (typically smaller) resulting data are written to the more accessible, but more costly [`Fast`](/scicomputing/store_posix/) storage.
+The scratch file system is maintained by SciComp for temporary storage of research data during active analysis.  This is a large, high-performance storage system.  It is not designed to be as available or as robust as the _home_ or _fast_ file systems.
 
+> Data on this platform is not backed up.  This storage is _not_ appropriate for storage of the primary or only copy of any data.
```

Comment:
Perhaps "not backed up" should be in bold.... (and 128 pt type! 😀 ).

Reply:
Man I miss the <blink> tag....